src/Routers/UsersRouter.js

```diff
@@ -29,6 +29,92 @@ export class UsersRouter extends ClassesRouter {
     }
   }
 
+  /**
+   * Validates a password request in login and verifyPassword
+   * @param {Object} req The request
+   * @returns {Object} User object
+   * @private
+   */
+  _handleValidatePassword(req) {
```

Comment:
we should rename to `_authenticateUserFromRequest(req)` what do you think?

THie method retuns the user, stripped of the private / hidden properties etc...

Reply:
No problem, I can rename it.